x-pack/plugin/core/src/test/java/org/elasticsearch/license/LicenseServiceTests.java

```diff
@@ -173,12 +262,7 @@ private void tryRegisterLicense(Settings baseSettings, License license, Consumer
             .put(DISCOVERY_TYPE_SETTING.getKey(), SINGLE_NODE_DISCOVERY_TYPE) // So we skip TLS checks
             .build();
 
-        final ClusterState clusterState = mock(ClusterState.class);
-        Mockito.when(clusterState.metadata()).thenReturn(Metadata.EMPTY_METADATA);
```

Comment:
We now also set the cluster name but that seems like a fair default and we get to re-use a bit of code